### PR TITLE
recieve -> receive

### PR DESCRIPTION
**Pull request by FunnyMan3595**
## What Does This PR Do
Fickses an mispelling.

## Why It's Good For The Game
Let's not confuse anyone who spells it correctly in their code.

## Testing
It compiles.

## Changelog
NPFC

### CODE REVIEW

**Review by AffectedArc07 — APPROVED** [review]
Approved my bit

**Review by Contrabang — APPROVED** [review]
the fact that this is not the first time we've had a pr for fixing these typos is funny

**Comment by GDNgit** [comment]
> the fact that this is not the first time we've had a pr for fixing these typos is funny

YOU COULD HAVE SAID `WE'VE RECEIVED A PR` GRRRR

File: code/modules/events/money_spam.dm [hunk 1]
```diff
@@ -78,7 +78,7 @@
 					"We are happy to inform you that due to the delay, we have been instructed to IMMEDIATELY deposit all funds into your account",\
 					"Dear fund beneficiary, We have please to inform you that overdue funds payment has finally been approved and released for payment",\
 					"Due to my lack of agents I require an off-world financial account to immediately deposit the sum of 1 POINT FIVE MILLION credits.",\
-					"Greetings sir, I regretfully to inform you that as I lay dying here due to my lack ofheirs I have chosen you to recieve the full sum of my lifetime savings of 1.5 billion credits")
+					"Greetings sir, I regretfully to inform you that as I lay dying here due to my lack ofheirs I have chosen you to receive the full sum of my lifetime savings of 1.5 billion credits")
```
Comment (Henri215) [inline code/modules/events/money_spam.dm:81]: If we go with the version that those are not intentional typos then its worth fixing "ofheirs" and the missing dot at the end too.

Comment (Contrabang) [reply code/modules/events/money_spam.dm:81]: i missed this, i think we should keep the poor spelling

Comment (Henri215) [reply code/modules/events/money_spam.dm:81]: If we keep it then a comment would be good, because this is the second PR trying to fix it 😆

**Review by Henri215 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]
maybe we should add CI for this at some point 😏

**Pull request merged**